packages/esm-patient-list-app/src/api/hooks.ts

```diff
@@ -144,3 +160,87 @@ export function useCohortTypes() {
   const swrResult = useSWR<FetchResponse<CohortResponse<CohortType>>, Error>(`${cohortUrl}/cohorttype`, openmrsFetch);
   return { ...swrResult, data: swrResult?.data?.data?.results };
 }
+
+export function usePatientListIdsForPatient(patientUuid: string) {
+  return useSWR('patientListIdsForPatient', async () => await getPatientListIdsForPatient(patientUuid));
```

Comment:
Instead of calling the `getPatientListIDsForPatient`, which itself makes a fetch request, can't we add the url and the `openmrsFetch` fetcher in the SWR?